app/assets/javascripts/main.js

```diff
@@ -44,7 +44,7 @@ const store = createStore(
  * @info(https://github.com/reactjs/react-router/tree/master/docs)
  * @type {Object}
  */
-const history = syncHistoryWithStore(hashHistory, store);
+// const history = syncHistoryWithStore(hashHistory, store);
```

Comment:
Delete would be better than commenting. We have git if we need to revive something like this.